### DISCUSSION TITLE
Advantage of container for NextDNS vs NextDNS native ubios/unifi support?

### DISCUSSION THREAD

**Discussion by seanb4t** [unifi-utilities/unifi-common#122] [Q&A] [answered]
The latest nextdns builds support installing on unifi/udm/ubios directly. The setup seems significantly less complex than the approach here - what are the advantages of going this route vs the 'native' from nextdns?

**Comment by boostchicken** [comment, marked as answer]
I would use native if possible. They support it! However, I think their means of achieving it is worse.
